go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/mattermost/mattermost-plugin-msteams
 
-go 1.21
-
-toolchain go1.21.6
```

Comment:
Can you elaborate on this change? Wondering if we want to follow the server pattern here which currently uses both directives?

Reply:
Sorry, I didn't change it manually, it looks like Google Mod Tidy did those changes.

Reply:
Reverting the change, the linter complain if I don't run the go mod tidy, and that is what is changing the go mod for that.